src/index.ts

```diff
@@ -466,7 +466,11 @@ export class UnleashClient extends TinyEmitter {
                         this.sdkState = 'healthy';
                     }
 
-                    if (!this.bootstrap && !this.readyEventEmitted) {
+                    if (
+                        (!this.bootstrap ||
+                            (this.bootstrap && !this.bootstrapOverride)) &&
+                        !this.readyEventEmitted
+                    ) {
```

Comment:
What would be the problem with just:
```suggestion
                    if (!this.readyEventEmitted) {
```

Reply:
This covers the missing condition, your suggestion makes sense afaict but it is broader.

Reply:
I simplified.